File: idakonyvtar/src/main/java/eu/eyan/idakonyvtar/model/ColumnConfigurations.java
package eu.eyan.idakonyvtar.model;

public enum ColumnConfigurations {
	MULTIFIELD("MultiMező"), AUTOCOMPLETE("AutoComplete"), MARC_CODE("MarcKód"), REMEMBERING(
			"Emlékező"), SHOW_IN_TABLE("Táblázatban");

	private String name;

	ColumnConfigurations(String configurationNév) {
		this.name = configurationNév;
	}

	public String getName() {
		return name;
	}
}

File: idakonyvtar/src/main/scala/eu/eyan/idakonyvtar/model/ColumnConfigurations.scala
package eu.eyan.idakonyvtar.model

object ColumnConfigurations extends Enumeration {

  type ColumnConfigurations = ColumnConfigurationsValue

  case class ColumnConfigurationsValue(name: String) extends Val(name)

  val MULTIFIELD = columnConfigurationsInternalValue("MultiMező")
  val AUTOCOMPLETE = columnConfigurationsInternalValue("AutoComplete")
  val MARC_CODE = columnConfigurationsInternalValue("MarcKód")
  val REMEMBERING = columnConfigurationsInternalValue("Emlékező")
  val SHOW_IN_TABLE = columnConfigurationsInternalValue("Táblázatban")

  protected final def columnConfigurationsInternalValue(name: String): ColumnConfigurationsValue = ColumnConfigurationsValue(name)
}